index.html

```diff
@@ -5,17 +5,39 @@
 <!--[if gt IE 8]>      <html class="no-js"> < !--<![endif]-->
 <html lang="en">
     <head>
+
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <title>Is-haq Adnan's Portfolio</title>
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
         <link rel="stylesheet" href="styles.css">
     </head>
     <body>
         <!--[if lt IE 7]>
             <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
+        <header>
+            <nav>
+                <a class="nav__icon" href="#">&lt;AI&gt;</a>
+                
+                <span class="material-symbols-outlined burger">
+                    menu
+                    </span>
+            </nav>
+        </header>
```

Comment:
please, should I remove the nav tag containing the anchor and span tag or still leave it intact? which makes a nav tag outside and inside

Reply:
Thank you, please should I still leave the outer nav tag housing the anchor tag and the menu bar intact?

Reply:
Hello @AdnanOlarmmi you use `header` tag for both of them so you can remove the outer tag. so please make sure to use nav tag for hamburger menu instead of span tag.